local.properties

```diff
@@ -1,10 +1,8 @@
-## This file is automatically generated by Android Studio.
-# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
-#
-# This file should *NOT* be checked into Version Control Systems,
+## This file must *NOT* be checked into Version Control Systems,
 # as it contains information specific to your local configuration.
 #
 # Location of the SDK. This is only used by Gradle.
 # For customization when using a Version Control System, please read the
 # header note.
-sdk.dir=/Users/marcin.wisniewski/Library/Android/sdk
\ No newline at end of file
+#Wed Aug 08 19:33:40 CEST 2018
+sdk.dir=C\:\\Users\\Pc\\AppData\\Local\\Android\\sdk
```

Comment:
This file must *NOT* be checked into Version Control Systems, as it contains information specific to your local configuration. :) 

Update your .gitignore file

Reply:
Done